change/@fluentui-react-tree-c13347b6-7d03-4e17-9814-38c22c21bacd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31598&commitId=3d644dbe58b6f8d327c81a62ec59a291a9249c45&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>